nhrpd/netlink_arp.c

```diff
@@ -65,9 +65,9 @@ static void netlink_neigh_msg(struct nlmsghdr *msg, struct zbuf *zb)
 	struct nhrp_cache *c;
 	struct interface *ifp;
 	struct zbuf payload;
-	union sockunion addr;
+	union sockunion addr, lladdr;
 	size_t len;
-	char buf[SU_ADDRSTRLEN];
+	char buf[4][SU_ADDRSTRLEN];
 	int state;
 
 	ndm = znl_pull(zb, sizeof(*ndm));
```

Comment:
should not we memset (&lladdr) ?

Reply:
thanks for formatting changes.
just to be sure , can you reply to my comment, please?

Reply:
👍 I've added memset(&lladdr) in the same commit as formatting changes.